examples/05-Retrieval-Model.ipynb

```diff
@@ -61,7 +61,8 @@
     "### Learning objectives\n",
```

Comment:
<p>We could maybe set this text as a conclusion of the section <code>Building the Model</code> instead of a section header? And maybe add some sentences to introduce the motivation for computing the top-k evaluation. Something like:</p><p> </p><blockquote>To determine the exact accuracy of our trained retrieval model, we need to compute the similarity score between a given query and all possible candidates. The higher the score of the positive candidate, the more accurate the model. We can do this using the TopKEncoder model and the following section shows how to instantiate it using the RetrievalModelV2 API.</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>